lib/rules/no-test-support-import.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @fileOverview no importing of test files and no exporting in test files
+ */
+
+'use strict';
+
+const NO_IMPORT_MESSAGE =
+  'Do not import a file from test-support into production code, only into files located in test.';
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'disallow importing of "test-support" files in production code.',
+      category: 'Testing',
+      recommended: true,
```

Comment:
We can't add things to `recommended` without a major version bump, can you change this to `false` for the time being?

Reply:
changed this to false.